.github/workflows/flintlock-ubuntu-perftest.yml

```diff
@@ -1,14 +1,14 @@
 name: Ubuntu Perf Test - Build and release
 
-on:
-  pull_request:
-    paths:
-      - 'flintlock/perf-test/**'
-    branches: [main]
-  push:
-    paths:
-      - 'flintlock/perf-test/**'
-    branches: [main]
+on: {}
+  # pull_request:
```

Comment:
nit: can we get rid of these comments?

Reply:
these test the base-ubuntu image, which is being removed in #52, so i will add removal of the perf image too